### PR TITLE
added gitVersion to version flag

### PR DESCRIPTION
**Pull request by nitinmewar**
Signed-off-by: nitin mewar <nitinmewar28@gmail.com>

added gitVersion to version flag.

### CODE REVIEW

**Comment by nitinmewar** [comment]
@jesseduffield please review this.

**Comment by jesseduffield** [comment]
Looks good to me. Just running tests now

File: pkg/app/entry_point.go [hunk 1]
```diff
@@ -268,3 +270,9 @@ func mergeBuildInfo(buildInfo *BuildInfo) {
 		buildInfo.Date = time.Value
 	}
 }
+func getGitVersionInfo() string {
```
Comment (jesseduffield) [inline pkg/app/entry_point.go:273]: We need a blank line between this function and the previous one

**Review by jesseduffield — CHANGES_REQUESTED** [review]
linting issue

**Comment by jesseduffield** [comment]
nice work @nitinmewar

**Comment by nitinmewar** [comment]
@jesseduffield thank you.

**Pull request merged**